scheduler/bin/run-docker.sh

```diff
@@ -33,6 +33,7 @@ docker run \
     --publish=${COOK_PORT}:${COOK_PORT} \
     -e "COOK_PORT=${COOK_PORT}" \
     -e "COOK_NREPL_PORT=${COOK_NREPL_PORT}" \
+    -e "COOK_FRAMEWORK_ID=cook-framework-${COOK_PORT}" \
```

Comment:
@wyegelwel I added back in the read from zookeeper such that if it's not configured, then it will attempt to read from zookeeper and use that one.